src/Illuminate/Mail/Markdown.php

```diff
@@ -69,9 +69,27 @@ public function render($view, array $data = [], $inliner = null)
         $contents = $bladeCompiler->usingEchoFormat(
             'new \Illuminate\Support\EncodedHtmlString(%s)',
             function () use ($view, $data) {
-                return $this->view->replaceNamespace(
-                    'mail', $this->htmlComponentPaths()
-                )->make($view, $data)->render();
+                EncodedHtmlString::encodeUsing(function ($value) {
```

Comment:
You could change this to an arrow function to get rid of the `use`:
```suggestion
fn () => EncodedHtmlString::encodeUsing(function ($value) {
```